docs/source/upgrade_notes.rst

```diff
@@ -3,10 +3,26 @@
 Upgrade Notes
 =============
 
+.. _ref-upgrade-notes-v3-5:
+
+|st2| v3.5
+----------
+
+* API will now set ``secure`` and ``samesite=lax`` cookie attributes for then auth cookie which
+  is set when authenticating via auth token / API key in query parameter (this approach is
+  primarily used by st2web).
+
+  If for some reason you want to change those default values, you can do that using
```

Comment:
Might read better saying "If you need to change those default values"?

Reply:
Thanks - changed.